queries.yaml

```diff
@@ -22,7 +22,7 @@ sources:
     template: ne.jinja2
   osmdata:
     template: osmdata.jinja2
-    start_zoom: 9
+    start_zoom: 8
```

Comment:
I think this "just works"... but for general OpenStreetMap data we're adding NE-OSM transition logic back to the JINJA because the min_zoom between NE and OSM overlap for the same features.

See related: https://github.com/tilezen/vector-datasource/pull/1359/files#r130750942